index.html

```diff
@@ -76,6 +76,40 @@ <h2>A project for first-time contributions</h2>
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
+        
+            <!-- ________ Wangari's card START ________  -->
```

Comment:
Be sure to run `npm run prettier-html` on your code so that our formatting is applied and helps align your code with the other contributors.